Bound Lua numbers and stack sizes for the Ditoo runtime.

--- a/luaconf.h
+++ b/luaconf.h
@@ -122,7 +122,7 @@
 /*
 @@ LUA_32BITS enables Lua with 32-bit integers and 32-bit floats.
 */
-#define LUA_32BITS	0
+#define LUA_32BITS	1
 
 
 /*
@@ -746,9 +746,9 @@
 ** (It must fit into max(size_t)/32 and max(int)/2.)
 */
 #if LUAI_IS32INT
-#define LUAI_MAXSTACK		1000000
+#define LUAI_MAXSTACK		512
 #else
-#define LUAI_MAXSTACK		15000
+#define LUAI_MAXSTACK		512
 #endif
 
 
